src/main/java/uk/gov/hmcts/reform/sscs/model/ReferenceData.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.hmcts.reform.sscs.model;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import lombok.extern.slf4j.Slf4j;
+import uk.gov.hmcts.reform.sscs.exception.ReferenceDataImportException;
+import uk.gov.hmcts.reform.sscs.utility.JsonDataReader;
+
+@Slf4j
+public abstract class ReferenceData {

```

Comment:
To me this could do with being a helper class wherein getReferenceData and generateHashmap are in that class. HearingDuration and SessionCategoryMap would then implement an interface called 'Hashable' or something and have the hash on them. Not 100% this will work, just want thoughts